applications/nrf_desktop/README.rst

```diff
@@ -62,6 +62,169 @@ The project contains configuration files for following boards:
 The application was designed to allow easy porting to a new HW.
 Check :ref:`porting_guide` for information about the device porting steps.
 
+User interface
+**************
+
+The nRF Desktop devices provide the user input to the host in the same way as other mice and keyboards, using the following options:
+
+    * Connection through USB
+    * Connection through BLE
+
+The devices can also provide input through the nRF Desktop dongle, which is connected to the host through USB.
+The dongle receives data from the peripherals connected through Bluetooth and forwards the data to the host.
+
+The peripheral connection can be based on standard BLE connection parameters or BLE with Low Latency Packet Mode (LLPM).
+LLPM is a proprietary Bluetooth extension from Nordic Semiconductor and can be used only if it is supported by both connected devices (desktop mice do not support it).
+LLPM enables sending data with high report rate (up to 1000 reports per second), which is not supported by the standard BLE.
+
+Connection through USB
+----------------------
+
+The nRF Desktop devices use the USB HID class.
+No additional software or drivers are required.
+
+The devices that are connected both wirelessly and through the USB at the same time will provide the input only through the USB.
+After the USB disconnection, the device will automatically switch to sending the data wirelessly using Bluetooth.
+
+Battery-powered devices with rechargeable batteries are charged through the USB.
+
+Connection through BLE
+----------------------
+
+When turned on, the nRF Desktop peripherals are advertising until they go to the suspended state or connect through Bluetooth.
+The peripheral supports one wireless connection at a time, but it may be bonded with multiple peers.
+
+The nRF Desktop central scans for all bonded peripherals that are not connected.
```

Comment:
I think Central should be capitalized in this context, let me know what you think @greg-fer

Reply:
I checked other BLE modules/samples, and peripheral was de-capitalized there. So I applied the same rule for Central.